### PR TITLE
Add papercut-proof gloves for the HoP

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Randomly thought of this funny gimmick item for HoP that has minor slash resistance (very minor, wont impact gameplay).

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image
image
image

**Changelog**

:cl: Ubaser
- add: The HoP now starts with a pair of papercut-proof gloves.

### CODE REVIEW

File: Resources/Textures/Clothing/Hands/Gloves/hop.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Drawn by Ubaser",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
```
Comment (deltanedas) [inline Resources/Textures/Clothing/Hands/Gloves/hop.rsi/meta.json:11]: .png pro

**Comment by VasilisThePikachu** [comment]
It looks like your images are corrupted somehow, they don't load even if I download the files

**Comment by TurboTrackerss14** [comment]
image
I was able to open the current images. 
Strange artifact visible on equiphand (the gloves are like 1% opacity in the centre of the image)

**Comment by UbaserB** [comment]
i legitimately have no idea what has been going on with my images, didnt use to do this before but im going to try my best to resolve this. Pretty sure its an issue of exporting from my drawing app.

**Comment by Emisse** [comment]
i think you should remove the spare set of captain gloves if thats an issue

**Comment by UbaserB** [comment]
> i think you should remove the spare set of captain gloves if thats an issue

it was a side not, not the main reason why i wanted to add these.

**Comment by Emisse** [comment]
ok but do it anyway

**Comment by UbaserB** [comment]
> ok but do it anyway

ok. same pr or different pr?

**Comment by Emisse** [comment]
diff

**Comment by TurboTrackerss14** [comment]
> i think you should remove the spare set of captain gloves if thats an issue

I thought there was already a discussion about leaving filledlocker contents alone because of replacement heads, theft, disguise?
also handing out cap spare gloves etc is powergaming so admin issue, no?

**Comment by Emisse** [comment]
putting more administrative work in the game is dumb and leads to admin burnout, reducing it through mechanical means is ideal

**Comment by UbaserB** [comment]
also i figured out the issue with the sprite erroring i will ping you today when i get the free time to update it and make a pr to remove spare cap gloves.

**Comment by Subversionary** [comment]
> handing out cap spare gloves etc is powergaming so admin issue

They're literally reskinned insuls with an only caveat of knowing where it came from. Even if it is somehow an admin issue it's not enforced.

**Pull request merged**